packages/env/lib/env.js

```diff
@@ -390,6 +390,14 @@ async function configureWordPress( environment, config ) {
 			options
 		);
 	}
+
+	for ( const [ key, value ] of Object.entries( config.config ) ) {
```

Comment:
We _maybe_ need to set config before activating plugins in case a plugin requires a constant in its activation hook.

Reply:
Yes